tests/nnapi/nnapi_gtest.skip.x86_64-linux.cpu

```diff
@@ -1,6 +1,4 @@
 GeneratedTests.abs_
-GeneratedTests.add_broadcast_quant8
-GeneratedTests.add_quant8
```

Comment:
Please remove these tests from `nnapi_gtest.skip.aarch64-linux.cpu` and `nnapi_gtest.skip.armv7l-linux.cpu`

Reply:
Removed the tests, thanks for the suggestion.